app/javascript/util/accountUtil.tsx

```diff
@@ -0,0 +1,9 @@
+// We will strictly validate the email address using the following regex.
+// Devise will also validate the email address on the backend with a looser pattern.
+export const emailRegex = new RegExp(
```

Comment:
do we need this? is it used?

Reply:
jk I found it!! curious why it has to be diff from the one we can import from bloom?

Reply:
That RegEx was the one that is currently being used in AngularJS, definitely open to using the Bloom one tho! For more info on this decision, here is the Slack thread: https://exygy.slack.com/archives/C030EGQAKK2/p1722363802571229